src/content/getinvolved.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Get Involved"
+description = "Actions you can take to help support and grow the Chapel project"
+
++++
+
+## Interested in helping the Chapel project grow?
+
+Here are some ways to get involved and help nurture the Chapel
+community:
+{.content-paragraph}
+
+
+### Easy Actions
+
+- Star the project [on GitHub](https://github.com/chapel-lang/chapel)
+- Follow us on [social media](/socials)
+- Subscribe to our [YouTube channel](https://www.youtube.com/@ChapelLanguage)
+- Like, repost, and amplify Chapel content you appreciate
+- Attend weekly project and/or deep-dive [meetings](/meetings)
+- Attend the monthly educators [meeting](/meetings)
```

Comment:
These two links go to the same place, which might be confusing. Perhaps we need to link to sub-headings within the page?